lib/ContentTypeParser.js

```diff
@@ -48,7 +48,8 @@ ContentTypeParser.prototype.getHandler = function (contentType) {
 ContentTypeParser.prototype.run = function (contentType, handler, request, reply) {
   var result = this.getHandler(contentType)(request.raw, done)
   if (result && typeof result.then === 'function') {
-    result.then(body => done(null, body)).catch(err => done(err, null))
+    result.then(body => done(null, body))
+      .catch(err => done(err, null))
```

Comment:
I think we can move to .then(func, func) signature here.

Reply:
Done